flowmachine/flowmachine/features/subscriber/interevent_period.py

```diff
@@ -0,0 +1,143 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Intervent period statistics, such as the average and standard deviation of the
+duration between calls.
+"""
+import warnings
+from typing import List
+
+from ...utils.utils import verify_columns_exist_in_all_tables
+from ..utilities import EventsTablesUnion
+from .metaclasses import SubscriberFeature
+
+valid_stats = {"count", "sum", "avg", "max", "min", "median", "stddev", "variance"}
+
+
```

Comment:
This seems to be from a different PR?

Reply:
I was building on top of previous branches, so it'd be better to approve them in order.